core/src/test/kotlin/org/stellar/anchor/auth/AuthHelperTest.kt

```diff
@@ -4,38 +4,34 @@ import io.mockk.*
 import java.util.*
 import kotlin.test.assertEquals
 import kotlin.test.assertNull
-import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.parallel.Execution
+import org.junit.jupiter.api.parallel.ExecutionMode.SAME_THREAD
 import org.junit.jupiter.params.ParameterizedTest
 import org.junit.jupiter.params.provider.EnumSource
-import org.stellar.anchor.auth.ApiAuthJwt.CallbackAuthJwt
-import org.stellar.anchor.auth.ApiAuthJwt.CustodyAuthJwt
-import org.stellar.anchor.auth.ApiAuthJwt.PlatformAuthJwt
+import org.stellar.anchor.auth.ApiAuthJwt.*
 import org.stellar.anchor.auth.AuthType.*
 import org.stellar.anchor.util.AuthHeader
 
+@Execution(SAME_THREAD)
```

Comment:
Does this override `junit.jupiter.execution.parallel.mode.default`?

Reply:
Yes it does.